rasa/core/policies/fallback.py

```diff
@@ -57,6 +58,13 @@ def __init__(
         self.core_threshold = core_threshold
         self.fallback_action_name = fallback_action_name
 
+        common_utils.raise_warning(
+            f"'{self.__class__.__name__}' is deprecated and will be removed in version "
+            "3.0. It is recommended to use the 'RulePolicy' instead.",
```

Comment:
are we really going to drag old style up to version `3`?

Reply:
We can also just say "in the future"?

Reply:
yes, let's do that